tools/fw-logger/rs-fw-logger.cpp

```diff
@@ -46,17 +46,23 @@ string datetime_string()
 
 int main(int argc, char* argv[])
 {
+    int default_sleep_ms = 25;
     CmdLine cmd("librealsense rs-fw-logger example tool", ' ', RS2_API_VERSION_STR);
     ValueArg<string> xml_arg("l", "load", "Full file path of HW Logger Events XML file", false, "", "Load HW Logger Events XML file");
+    ValueArg<int> sleep_arg("s", "sleep", "Sleep between each log message pulling (in milliseconds)", false, default_sleep_ms, "");
     SwitchArg flash_logs_arg("f", "flash", "Flash Logs Request", false);
     cmd.add(xml_arg); 
+    cmd.add(sleep_arg);
     cmd.add(flash_logs_arg);
     cmd.parse(argc, argv);
 
     log_to_file(RS2_LOG_SEVERITY_WARN, "librealsense.log");
 
     auto use_xml_file = false;
     auto xml_full_file_path = xml_arg.getValue();
+    auto sleep_ms = sleep_arg.getValue();
+    if (sleep_ms < 1 || sleep_ms > 200)
```

Comment:
Also print user notification to console to explain why the parameter is changed

Reply:
done